pkg/cover/backend/backend.go

```diff
@@ -64,7 +65,7 @@ type Range struct {
 const LineEnd = 1 << 30
 
 func Make(target *targets.Target, vm, objDir, srcDir, buildDir string,
-	moduleObj []string, modules []host.KernelModule) (*Impl, error) {
+	moduleObj []string, modules []*host.KernelModule) (*Impl, error) {
```

Comment:
Please move this switch to passing pointers to a separate commit. It adds lots of changes lines here and there intermixed few important logical changes, which a re much harder to review in this large commit.